src/Assets/ugui-mvvm/Editor/CommandBindingEditor.cs

```diff
@@ -78,19 +77,13 @@ void OnEnable()
 
     public override void OnInspectorGUI()
     {
-        // Only update the focused element name during the Layout event, since all controls must be static between Layout & Repaint.
-        if (Event.current.type == EventType.Layout)
-        {
-            _focusedControl = GUI.GetNameOfFocusedControl();
-        }
-
         serializedObject.Update();
 
         EditorGUILayout.PropertyField(_vprop);
         if (_vprop.objectReferenceValue != null)
             INPCBindingEditor.DrawComponentEvents(_vprop, _veprop);
 
-        INPCBindingEditor.DrawCRefProp(serializedObject.targetObject.GetInstanceID(), _focusedControl, _vmprop, GUIContent.none, typeof(ICommand));
+        EditorGUILayout.PropertyField(_vmprop);
 
```

Comment:
As a next step, can we add validation of "ICommand" based properties as the only set of suggestions that show up for the CommandBinding properties ?

Reply:
We'd have to think about when to do this validation.  Probably only when focus went away.  Because you'd want to show non-ICommand properties in case they had subproperties which _were_ ICommands.  But yes, I think this is a good idea.

Another potential improvement that I thought about is to show the type of each item in the list.